packages/select/src/FilterMultiSelect/components/ListBoxSection/ListBoxSection.tsx

```diff
@@ -1,24 +1,61 @@
-import React from "react"
+import React, { ReactNode, useState, useEffect } from "react"
+import { v4 } from "uuid"
+import { VisuallyHidden } from "@kaizen/a11y"
 import { MultiSelectItem } from "../../../types"
 import styles from "./ListBoxSection.module.scss"
-
 export interface ListBoxSectionProps {
   items: MultiSelectItem[]
   /**
    * Becomes an aria-label on the section, informing
    * unsighted users
    */
   sectionName: string
+  /** If provided, will override the aria-label for this group */
+  sectionHeader?: ReactNode
   children: (item: MultiSelectItem) => React.ReactNode
 }
 
 export const ListBoxSection = ({
   items,
   sectionName,
   children,
-}: ListBoxSectionProps): JSX.Element => (
-  <div aria-label={sectionName} className={styles.listBoxSection}>
-    {Array.from(items).map(node => node != undefined && children(node))}
-  </div>
-)
+  sectionHeader,
+}: ListBoxSectionProps): JSX.Element => {
+  const [listSectionId, setListSectionId] = useState<string | undefined>(
+    undefined
+  )
+
+  useEffect(() => {
+    setListSectionId(v4())
```

Comment:
You can just do this in the initial state declaration, no need for the useEffect

Reply:
Ah right, you mean like:
```  const [listSectionId] = useState<string>(v4())```

Reply:
Yup, you should see that used around Kaizen here and there